### PR TITLE
Inline Editable Cell Cleanup

### PR DESCRIPTION
**Pull request by nickwmurphy**
All Inline Editable fields will now follow this convention:

inline-fields

### CODE REVIEW

File: table/README.md [hunk 1]
```diff
@@ -4,6 +4,9 @@ A table component with self-managed checkboxes, sticky head, sticky right panel,
 
 # Change Log
 
+##### 4.3.1
```
Comment (ben-burlingham) [inline table/README.md]: These changes will break style overrides so IMO they should be `major`, not `patch`

**Comment by ben-burlingham** [comment]
Can we extend these conventions to `date-select`, `select`, and `filters`?

File: table/README.md (hunk 1 above)
Comment (nickwmurphy) [reply table/README.md]: I agree. I will bump to 5.0.0.

File: table/sandbox/sandbox.scss [hunk 2]
```diff
@@ -1,5 +1,5 @@
 .sandboxContainer {
-  font: 11px sans-serif;
+  font: 11px Lato, sans-serif;
```
Comment (ben-burlingham) [inline table/sandbox/sandbox.scss]: nb No Lato in sandbox

**Pull request merged**